products/sle/main.pm

```diff
@@ -820,6 +820,7 @@ elsif (get_var("QAM_MINIMAL")) {
     loadtest "qam-minimal/install_update";
     loadtest "qam-minimal/update_minimal";
     loadtest "qam-minimal/check_logs";
+    loadtest 'qam-minimal/s390tools' if check_var('ARCH', 's390x');
```

Comment:
if you use `check_var('ARCH', 's390x')`, those tests will also be executed on s390x KVM - we e.g. dont have DASDs there, so I think not all of those tests are working there